web/src/stream_edit_subscribers.ts

```diff
@@ -183,13 +224,14 @@ function subscribe_new_users({pill_user_ids}) {
 
     const user_ids = [...user_id_set];
 
-    function invite_success(data) {
+    function invite_success(data: unknown): void {
+        const clean_data = add_user_ids_api_response_schema.parse(data);
```

Comment:
As pointed out by the core maintainers couple of time, this is not the most correct/healthy naming convention. This [CZO comment](https://chat.zulip.org/#narrow/stream/6-frontend/topic/typescript.20migration/near/1554084) explains why. I have fixed this and couple other such cases in [#30521 ](https://github.com/zulip/zulip/pull/30521)